tests/docstrings_examples/DocTest.res

```diff
@@ -158,142 +153,45 @@ let extractExamples = async () => {
   examples
 }
 
-let compileTest = async (~code) => {
-  // NOTE: warnings argument (-w) should be before eval (-e) argument
-  let args = ["-w", "-3-109-44", "-e", code]
-  let {stderr, stdout} = await SpawnAsync.run(~command=bscBin, ~args)
-
-  stderr->Array.length > 0 ? Error(stderr->getOutput) : Ok(stdout->getOutput)
-}
-
-let compileExamples = async examples => {
-  Console.log(`Compiling ${examples->Array.length->Int.toString} examples from docstrings...`)
-
-  let compiled = []
-  let compilationErrors = []
-
-  await examples->ArrayUtils.forEachAsyncInBatches(~batchSize, async example => {
-    // let id = example.id->String.replaceAll(".", "__")
-    let rescriptCode = example->getCodeBlocks
-
-    switch await compileTest(~code=rescriptCode) {
-    | Ok(jsCode) => compiled->Array.push((example, rescriptCode, jsCode))
-    | Error(err) => compilationErrors->Array.push((example, ReScriptError(err)))
-    }
-  })
-
-  (compiled, compilationErrors)
-}
-
-let runTest = async code => {
-  let {stdout, stderr, code: exitCode} = await SpawnAsync.run(
-    ~command="node",
-    ~args=["-e", code, "--input-type", "commonjs"],
-    ~options={cwd: Process.cwd(), timeout: 2000},
+let main = async () => {
+  let examples = await extractExamples()
+  examples->Array.sort((a, b) =>
+    Obj.magic(a.id) > Obj.magic(b.id) ? Ordering.fromInt(1) : Ordering.fromInt(-1)
```

Comment:
Why do we need `Obj.magic` here?

I think this can just be
```rescript
  examples->Array.sort((a, b) => String.compare(a.id, b.id))
```
and should be moved into the `extractExamples` function.

Reply:
Thanks